server/protocols/webadmin/webadmin-mailbox/src/test/java/org/apache/james/webadmin/routes/UserQuotaRoutesTest.java

```diff
@@ -439,8 +468,8 @@ public void getQuotaShouldReturnBothWhenNoSize() throws Exception {
                 .extract()
                 .jsonPath();
 
-        assertThat(jsonPath.getObject(SIZE, Long.class)).isNull();
-        assertThat(jsonPath.getLong(COUNT)).isEqualTo(maxMessage);
+        assertThat(jsonPath.getObject("user." + SIZE, Long.class)).isNull();
+        assertThat(jsonPath.getLong("user." + COUNT)).isEqualTo(maxMessage);
     }
 
```

Comment:
IMO we are missing at least one test.

How does the `GET` answer looks like when we don't have user specific quota specifications

Reply:
ok, added